docs/roman_datamodels/datamodels/using_datamodels.rst

```diff
@@ -3,6 +3,14 @@
 Using Roman Datamodels
 ======================
 
+.. note::
+   IPython and jupyter won't automatically tab-complete the
+   dynamic attributes of datamodels. To enable tab-completion
+   add the following to your `ipython configuration <https://ipython.readthedocs.io/en/stable/config/intro.html>`_::
+
+       c.IPCompleter.evaluation = "unsafe"
+       c.IPCompleter.use_jedi = False
+
```

Comment:
This should not be a blanket recommendation. It will break the `jedi` language server which is much more fully featured than the basic one that the repl uses. For example turning this off will mean that you loose some of the static code analysis on object that exist prior to instantiation and it will stop iPython/Jupyter from doing any static type analysis which may surprise users.